include/pacmod_game_control/globals.h

```diff
@@ -17,10 +17,6 @@
 #include <std_msgs/Bool.h>
 #include <pacmod_msgs/VehicleSpeedRpt.h>
 
-namespace AS
-{
-namespace Joystick
-{
 enum GamepadType
```

Comment:
use enum class GamepadType or enum GamepadType : uint8_t and give value to each entry.